apps/avatax/src/pages/api/webhooks/order-confirmed.ts

```diff
@@ -72,23 +73,37 @@ export default wrapWithLoggerContext(
         logger.debug("Confirming order...");
 
         if (taxProviderResult.isOk()) {
-          const confirmedOrder = await taxProviderResult.value.confirmOrder(payload.order);
-
-          logger.info("Order confirmed", { confirmedOrder });
-          const client = createInstrumentedGraphqlClient({
-            saleorApiUrl,
-            token,
-          });
-
-          const orderMetadataManager = new OrderMetadataManager(client);
-
-          await orderMetadataManager.updateOrderMetadataWithExternalId(
-            payload.order.id,
-            confirmedOrder.id,
-          );
-          logger.info("Updated order metadata with externalId");
+          try {
+            const confirmedOrder = await taxProviderResult.value.confirmOrder(payload.order);
+
+            logger.info("Order confirmed", { confirmedOrder });
```

Comment:
question: is it ok, if we log entire order object? we don't want to log PII, especially about customers 😅

Reply:
`confirmedOrder` has only `id`, but just to be safe I'll make it more explicit.